tests/dash/gnmi_utils.py

```diff
@@ -324,6 +326,41 @@ def gnmi_get(duthost, ptfhost, path_list):
             raise Exception("error:" + msg)
 
 
+def apply_messages(
+    localhost,
+    duthost,
+    ptfhost,
+    messages,
+    dpu_index,
+    set=True,
+    wait_after_apply=5,
+    max_updates_in_single_cmd=1024,
+):
+    env = GNMIEnvironment(duthost)
+    update_list = []
+    delete_list = []
+    for i, (key, config_dict) in enumerate(messages.items()):
+        message = proto_utils.parse_dash_proto(key, config_dict)
+        keys = key.split(":", 1)
+        gnmi_key = keys[0] + "[key=" + keys[1] + "]"
+        filename = f"update{i}"
+
+        if set:
+            if proto_utils.ENABLE_PROTO:
+                path = f"/APPL_DB/dpu{dpu_index}/{gnmi_key}:$/root/{filename}"
+            else:
+                path = f"/APPL_DB/dpu{dpu_index}/{gnmi_key}:@/root/{filename}"
```

Comment:
The else seems to be the same as the if. Should it be different or the if else is not needed?